src/Microsoft.CodeQuality.Analyzers.Exp/UnitTests/Design/ValidateArgumentsOfPublicMethodsTests.cs

```diff
@@ -39,9 +40,9 @@ public int M1(C c)
         return c.X;
     }
 }
-");
+").ConfigureAwait(false);
```

Comment:
:bulb: Probably a good idea to turn off this analyzer in the test projects

Reply:
CA2007 is now disabled for test projects, and the change to add this was reverted during a rebase.